vcs/gogpm.go

```diff
@@ -73,7 +74,30 @@ func (p *PackageRepo) RootImportPath() string {
 }
 
 func (p *PackageRepo) Dir() string {
-	return filepath.Join(os.Getenv("GOPATH"), "src", p.rr.root)
+	// split gopath
+	goPath := os.Getenv("GOPATH")
+	paths := strings.Split(goPath, ":")
```

Comment:
Hi,

This is not ideal.

Instead of `strings.Split` you should use [`filepath.SplitList`](https://godoc.org/path/filepath#SplitList) which is made for this purpose. The reason is that ":" is only going to work on some systems, but not all:

> The GOPATH environment variable lists places to look for Go code. On Unix, the value is a colon-separated string. On Windows, the value is a semicolon-separated string. On Plan 9, the value is a list.